docs/core/extensions/data-classification.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Data classification in .NET
+description: Learn how to use .NET data classification libraries to categorize your application's data.
+ms.date: 03/20/2025
+---
+
+# Data classification in .NET
+
+Data classification helps you categorize (or classify) data based on its sensitivity and protection level. The <xref:Microsoft.Extensions.Compliance.Classification.DataClassification> structure lets you label sensitive information and enforce policies based on these labels.
+
+- <xref:Microsoft.Extensions.Compliance.Classification.DataClassification.TaxonomyName?displayProperty=nameWithType>: Identifies the classification system.
+- <xref:Microsoft.Extensions.Compliance.Classification.DataClassification.Value?displayProperty=nameWithType>: Represents the specific label within the taxonomy.
+
+In some situations, you might need to specify that data explicitly has no data classification, this is achieved with <xref:Microsoft.Extensions.Compliance.Classification.DataClassification.None?displayProperty=nameWithType>. Similarly, you might need to specify that data classification is unknown—use <xref:Microsoft.Extensions.Compliance.Classification.DataClassification.Unknown?displayProperty=nameWithType> in these cases.
+
+## Install classification package
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.Compliance.Classification
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.Extensions.Compliance.Classification"
+                    Version="*" />
+</ItemGroup>
+```
+
+---
+
+## Create custom classifications
+
+Define custom classifications by creating static members for different types of sensitive data. This gives you a consistent way to label and handle data across your app. Consider the following example class:
+
+```csharp
+using Microsoft.Extensions.Compliance.Classification;
+
+internal static class MyTaxonomyClassifications
+{
+    internal static string Name => "MyTaxonomy";
+
+    internal static DataClassification PrivateInformation => new(Name, nameof(PrivateInformation));
+    internal static DataClassification CreditCardNumber => new(Name, nameof(CreditCardNumber));
+    internal static DataClassification SocialSecurityNumber => new(Name, nameof(SocialSecurityNumber));
+}
+```
+
+If you want to share your custom classification taxonomy among your applications, this class should be public. For example, you can have a shared library containing custom classifications, that you can use in multiple applications.
```

Comment:
```suggestion
If you want to share your custom classification taxonomy with other apps, this class and its members should be `public` instead of `internal`. For example, you can have a shared library containing custom classifications, that you can use in multiple applications.
```